test/EstateRegistry.js

```diff
@@ -251,6 +247,19 @@ contract('EstateRegistry', accounts => {
       })
     })
 
+    it('should allow operator to create an estate', async function() {
+      await land.assignMultipleParcels([0], [1], user, sentByCreator)
+      await land.setApprovalForAll(anotherUser, true, sentByUser)
+      await createEstate([0], [1], user, sentByAnotherUser)
+      await assertEstateCount(user, 1)
+    })
+
+    it('fails if sender is not owner or operator of all lands', async function() {
+      await land.assignMultipleParcels([0, 0], [1, 2], user, sentByCreator)
+      await land.approve(anotherUser, 1, sentByUser)
+      await assertRevert(createEstate([0, 0], [1, 2], user, sentByAnotherUser))
```

Comment:
I think it's clearer if we use:

`await assertRevert(createEstate([0], [1], anotherUser, sentByAnotherUser))`

To make it clear that it is someone else trying to steal the ownership of LAND to himself.

Reply:
Do you mean to change `user` to `anotherUser` as the beneficiary, right?

Reply:
Yes